scripts/release/build_esm.js

```diff
@@ -1,32 +1,23 @@
-import path from 'path'
-import fs from 'fs'
 import { execSync } from 'child_process'
 
 const exec = command => execSync(command, { stdio: 'inherit' })
 
-const svgrTempDir = path.resolve(process.cwd(), 'tmp/svgr')
-const svgrAssetsDir = path.resolve(svgrTempDir, 'assets')
-const svgrComponentsDir = path.resolve(svgrTempDir, 'components')
-
-function ensureDir(dir) {
-  if (!fs.existsSync(dir)) {
-    fs.mkdirSync(dir)
-  }
-}
-
-// svg will complain if the --out-dir does not exist
```

Comment:
apparently that doesn't seem to be the case... not sure why I encountered it originally